gsa/src/gmp/models/scanner.js

```diff
@@ -92,6 +92,50 @@ const parse_scanner_info = (info = {}) => {
 class Scanner extends Model {
   static entityType = 'scanner';
 
+  static parseObject(object) {
+    const copy = super.parseObject(object);
+
+    copy.scannerType = scannerTypeInt(object.type);
```

Comment:
I agree with this. However, we might want to wait until we convert the corresponding entities (scanner, scanconfig and probably some more) to Hyperion completely. We might get problems in the components that still run with gsad otherwise.

Reply:
Oh yeah right. forgot about other pages still running with gsad.